components/Features/FeatureFormSection.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import styles from './FeatureFormSection.module.scss';
+import FeatureForm from './FeatureForm';
+
+export default function FeatureFormSection() {
+  return (
+    <div id="Form-section" className={styles['form-container']}>
+      <div className={styles['form-text-container']}>
+        <div className={styles['form-text-wrapper']}>
+          <div className={styles['text-container']}>
+            <h1 className={styles.header}>Marketing and Publishing</h1>
+            <p className={styles.paragraph}>
```

Comment:
Same here, try looking through Semantic components to see what you can replace: https://react.semantic-ui.com